specifyweb/frontend/js_src/lib/components/Molecules/ResourceLink.tsx

```diff
@@ -25,6 +25,7 @@ export function ResourceLink<COMPONENT extends typeof Link['Icon']>({
   props: rawProps,
   resourceView,
   children,
+  autoClose,
```

Comment:
```suggestion
  autoClose = false,
```

whenever possible, try to set the default value when prop is initialized, rather than when used.
this applies both to react components, and regular functions

benefits:
- some IDEs provide better integration and autocomplete when you do this (i.e when calling the function, they will show what the default value for an argument is)
  - similarly, someone looking at a component, will easily know what a default value for a prop is by looking at the component header (without having to search for it's usages in the code)
- variable is declared only once, but may be used multiple times - by setting default value, you avoid the need for specifying the default value/fallback place in every place where the variable is used (thus, saving code, making it easier to change the default value, and reducing possibly bugs when default values mismatch)

drawbacks:
- if default value is an object or other mutable value, declaring default value may recreate default value multiple times - in that case, you can define the default value outside the component, and use it as a variable (only applies to react components, not functions). see example solution - https://github.com/specify/specify7/pull/4357/files#diff-c703adeb4352c7832286317011ba686112a5f56f7c0ca20f3a493fc2561197d4R99-R107